newIDE/app/src/ExportAndShare/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -63,22 +65,68 @@ export default class LocalPreviewLauncher extends React.Component<
     hideMenuBar: true,
     alwaysOnTop: true,
     numberOfWindows: 1,
+    captureOptions: null,
   };
   _networkPreviewSubscriptionChecker: ?SubscriptionCheckerInterface = null;
   _hotReloadSubscriptionChecker: ?SubscriptionCheckerInterface = null;
 
   _openPreviewBrowserWindow = () => {
-    const { previewGamePath, previewBrowserWindowOptions } = this.state;
+    const {
+      previewGamePath,
+      previewBrowserWindowOptions,
+      captureOptions,
+      alwaysOnTop,
+      hideMenuBar,
+      numberOfWindows,
+    } = this.state;
     if (!previewBrowserWindowOptions || !previewGamePath) return;
 
     if (!ipcRenderer) return;
 
     ipcRenderer.invoke('preview-open', {
       previewBrowserWindowOptions,
       previewGameIndexHtmlPath: `file://${previewGamePath}/index.html`,
-      alwaysOnTop: this.state.alwaysOnTop,
-      hideMenuBar: this.state.hideMenuBar,
-      numberOfWindows: this.state.numberOfWindows,
+      alwaysOnTop,
+      hideMenuBar,
+      numberOfWindows,
+      captureOptions,
+    });
+
+    ipcRenderer.removeAllListeners('preview-window-closed');
```

Comment:
What if I launch two or 3 or more previews?

Reply:
This only works for the final preview launched indeed, I think that's fine?